### DISCUSSION TITLE
Heroic Games Launcher protocol?

### DISCUSSION THREAD

**Discussion by Qronikarz** [Heroic-Games-Launcher/HeroicGamesLauncher#3008] [General]
Hi.
I have a question because I didn't want to file an issue request before asking.

Does heroic have something like Steam's run link? For example - steam://run/000000/
You can see it on SteamDB and it allows you to run (or install) games from the page.
![steamdbrun](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/84787215/86e90849-0367-47c6-8691-64b3d9cb4e3d)

I'm currently in process of adding new features to my media tagging board on GitHub and figured that if I can make such a button for Steam then Heroic Games Launcher also could have it's own launch button. Question, is it already possible or do I need to fill an issue request for it?

**Comment by imLinguin** [comment]
It's possible here is the template: `heroic://launch/{runner}/{appName}`
where:
- `{runner}` - `gog`, `legendary` or `nile`
- `{appName}` - internal id of the game (can be obtained at the bottom of the game settings)

Runner is technically optional, but it's best to specify it

**Comment by Qronikarz** [comment]
Thank you very much. That seems sufficient for now.
